loongarch64: add kernel 6.12 support

hackpascal opened the pull request:
Allow selecting 6.12 as testing kernel on loongarch64.

Runtime-tested on Loongson-3A6000-HV-7A2000-XA61200 with AMD RX580

namiltd commented on target/linux/loongarch64/config-6.12:
```diff
@@ -0,0 +1,761 @@
+CONFIG_HDMI=y
+CONFIG_HIBERNATE_CALLBACKS=y
+CONFIG_HIBERNATION=y
+CONFIG_HIBERNATION_COMP_LZO=y
```
Add:
# CONFIG_HIBERNATION_COMP_LZ4 is not set

hackpascal replied:
It's weird this didn't show up in my local build.

hauke commented:
I merge the procd patch, please rebase.

Please use the `scripts/kernel_bump.sh` script to generate the kernel update.

hackpascal committed:
loongarch64: Restore kernel files for v6.6

This is an automatically generated commit which aids following Kernel patch
history, as git will see the move and copy as a rename thus defeating the
purpose.

For the original discussion see:
https://lists.openwrt.org/pipermail/openwrt-devel/2023-October/041673.html

Signed-off-by: Weijie Gao <hackpascal@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19136
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

hackpascal committed:
loongarch64: Create kernel files for v6.12 (from v6.6)

This is an automatically generated commit.

When doing `git bisect`, consider `git bisect --skip`.

Signed-off-by: Weijie Gao <hackpascal@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19136
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

hackpascal committed:
loongarch64: add 6.12 as testing kernel

Runtime-tested on Loongson-3A6000-HV-7A2000-XA61200 with AMD RX580

Signed-off-by: Weijie Gao <hackpascal@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19136
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
